### DISCUSSION TITLE
My Relationships Picker is not working on Table Display

### DISCUSSION THREAD

**Discussion by Winniepoom** [Budibase/budibase#11244] [Help]
I try to use picker to sort relationships data from my table, when I picker is not showing data in my table for example I want to sort 3D roles I pick it and it show  No rows found

**Comment by andz-bb** [comment]
hey @Winniepoom, could you share some details about how your tables are set up please? Which data source are you using and what is the data type of the job title id?

You'll also need to make sure that your Table block is using `Dataprovider` as the data source rather than directly accessing the table.

If you could send these details or an [app export](https://docs.budibase.com/docs/export-and-import-apps#export-an-app) and [db schema](https://docs.budibase.com/docs/troubleshooting-sql#exporting-your-schema), I can have a look at fixing the filter for your use case.
